perception/autoware_tensorrt_yolox/include/autoware/tensorrt_yolox/tensorrt_yolox.hpp

```diff
@@ -89,15 +89,22 @@ class TrtYoloX
     const std::string & model_path, const std::string & precision, const int num_class = 8,
     const float score_threshold = 0.3, const float nms_threshold = 0.7,
     const tensorrt_common::BuildConfig build_config = tensorrt_common::BuildConfig(),
-    const bool use_gpu_preprocess = false, std::string calibration_image_list_file = std::string(),
-    const double norm_factor = 1.0, [[maybe_unused]] const std::string & cache_dir = "",
+    const bool use_gpu_preprocess = false, const int gpu_id = 0,
```

Comment:
```suggestion
    const bool use_gpu_preprocess = false, const uint8_t gpu_id = 0,
```
Although int is frequently used in Autoware, it is actualy an unsafe type. Please use uint8_t instaed.
cf. https://matt.sh/howto-c#:~:text=Writing%20code-,Types,-If%20you%20find

Reply:
Done :+1:

Reply:
Done :+1: